docs/installation/native-installation.md

```diff
@@ -45,15 +45,28 @@ php phing build-demo-dev
 
 ## Run integrated HTTP server
 
-```sh
-php bin/console server:run
+First follow [installation instructions](https://symfony.com/doc/current/setup/symfony_server.html#installation) for the `symfony` binary which provides its own web server.
+
+Then you can run integrated webserver for the first domain:
+```bash
+symfony server:start
```

Comment:
Also, I think the best solution (might be the next interaction) to create a command that will provide a user with all available domains.

Reply:
I agree, but sadly I don't know a quick&nice™ way how to achieve this behavior now.

Reply:
We have some commands with options, but I do understand it might not be part of this US. Can you please create another one?

Reply:
Yup, created in our backlog.